client/src/pages/NumberQuestionPage.js

```diff
@@ -24,10 +25,11 @@ function NumberQuestionPage() {
   }, [id]);
 
   return (
-    <Numbermain>
-      <div>
-        <h1>Question {id}</h1>
-
+    <div>
+      <h1>Question {id}</h1>
+      <questionstyling>
+        <Counter number={incPoints} />
```

Comment:
Here you see, that number should be a `num`, not a `func` in your Counter.js

Reply:
Have implemented accordingly!